code/controllers/subsystem.dm

```diff
@@ -65,6 +65,9 @@
 
 	/// Tracks the current execution state of the subsystem. Used to handle subsystems that sleep in fire so the mc doesn't run them again while they are sleeping
 	var/state = SS_IDLE
+	
+	/// Tracks how many times a subsystem has ever slept in fire().
+	var/slept_count = 0
```

Comment:
can we log it? would be good to know this info as a metric to improve on, where it's not intended at least